src/simplay/src/simplay/core.py

```diff
@@ -19,14 +21,32 @@ class VisualComponent:
 
     :param env: The environment instance.
     :param id: The id of the component.
-    :param type: The type of the component, one of ``'RESOURCE'``, ``'CONTAINER'``, ``'STORE'``, ``'PROCESS'`` or ``'CUSTOM'``.
-    :param graphic: The graphic of the component, either a 'simple' visual or a collection of sprites. Must be registered in the ``VisualizationManager``.
-    :param tint: The tint of the component. This only works with visuals and sprites that have transparent pixels. The tint is applied to the pixels that are not transparent.
+    :param type: The type of the component, one of ``'RESOURCE'``,
+     ``'CONTAINER'``, ``'STORE'``, ``'PROCESS'`` or ``'CUSTOM'``.
+    :param graphic: The graphic of the component, either a 'simple' visual or
+     a collection of sprites. Must be registered in the `VisualizationManager`.
+    :param tint: The tint of the component. This only works with visuals and
```

Comment:
As I remember this should eb a hexcode in the style of 0x123fff right? if this is the case all documentations of tint need to mention this "format"